packages/teams-js/src/internal/constants.ts

```diff
@@ -190,3 +190,98 @@ export const errorRuntimeNotSupported = 'The runtime version is not supported';
 
 /** @hidden */
 export const errorCallNotStarted = 'The call was not properly started';
+
+/**
```

Comment:
How about call this file something like: TelemetryConstants.ts?

Reply:
I personally agree with that. Do you have any thoughts? @TrevorJoelHarris

Reply:
These should actually also go in `telemetry.ts`

Reply:
They are moved to telemetry.ts